llvm/lib/Target/SPIRV/SPIRVTargetMachine.cpp

```diff
@@ -164,6 +164,7 @@ void SPIRVPassConfig::addIRPasses() {
     //  - all loop exits are dominated by the loop pre-header.
     //  - loops have a single back-edge.
     addPass(createLoopSimplifyPass());
+    addPass(createSPIRVMergeRegionExitTargetsPass());
```

Comment:
Add another pass summary comment like the one above?

Reply:
done thanks!